jmxfetch.py

```diff
@@ -275,6 +275,7 @@ def _start(self, path_to_java, java_run_opts, jmx_checks, command, reporter, too
             log.info("Running %s" % " ".join(subprocess_args))
 
             # Launch JMXfetch subprocess
+            # FIXME: Use get_subprocess_output() instead of subprocess.Popen
```

Comment:
This one is gonna take some more work in the morning. This function returns the return code of the process, making the return of get_subprocess_output inadequate, unless the caller of this function is okay with either a 0 or 1 return code, representing either an error string that is either empty or not.